language/language_samples.rb

```diff
@@ -21,9 +21,11 @@ def sentiment_from_text project_id:, text_content:
 
   language  = Google::Cloud::Language.new project: project_id
   document  = language.document text_content
-  sentiment = document.sentiment
-
-  puts "#{sentiment.score} (#{sentiment.magnitude})"
+  puts "Overall document sentiment: (#{document.sentiment.score})"
```

Comment:
You can reduce the number of chained method calls by creating a new variable `sentiment = document.sentiment`.